Issue with CLion.

adding Path as dependency gives error in CLion.

> /Path.swift/Package.swift:15:67: error: use of unresolved identifier 'package'; did you mean 'Package'?
> ; import Foundation; print(String(data: try! JSONEncoder().encode(package), encoding: .utf8)!)
>                                                                   ^~~~~~~
>                                                                   Package
> PackageDescription.Package:1:20: note: 'Package' declared here
> final public class Package {
>                    ^

--- a/Package.swift
+++ b/Package.swift
@@ -1,7 +1,7 @@
 // swift-tools-version:4.2
 import PackageDescription
 
-let pkg = Package(
+let package = Package(
     name: "Path.swift",
     products: [
         .library(name: "Path", targets: ["Path"]),

